services/scan/src/store.ts

```diff
@@ -486,45 +645,49 @@ export class Store {
       return true;
     }
 
-    const scannerBackedUpAt = this.getBackupTimestamp(BackupKey.Scanner);
-
-    if (!this.batchStatus().length) {
+    // Allow unconfiguring if no ballots have been counted
+    if (!this.getBallotsCounted()) {
       return true;
     }
 
+    const scannerBackedUpAt = this.getScannerBackupTimestamp();
+
     // Require that a scanner backup has taken place
     if (!scannerBackedUpAt) {
       return false;
     }
 
-    const { maxCvrsCreatedAt, maxCvrsDeletedAt } = this.client.one(
-      'select max(created_at) as maxCvrsCreatedAt, max(deleted_at) as maxCvrsDeletedAt from sheets'
-    ) as {
-      maxCvrsCreatedAt: Iso8601Timestamp;
-      maxCvrsDeletedAt: Iso8601Timestamp;
+    // Adding or deleting sheets will update the CVR count
+    const { maxSheetsCreatedAt, maxSheetsDeletedAt } = this.client.one(`
+        select
+          max(created_at) as maxSheetsCreatedAt, 
+          max(deleted_at) as maxSheetsDeletedAt
+        from sheets
+      `) as {
+      maxSheetsCreatedAt: Iso8601Timestamp;
+      maxSheetsDeletedAt: Iso8601Timestamp;
     };
-    const { maxBatchesStartedAt, maxBatchesDeletedAt } = this.client.one(
-      'select max(started_at) as maxBatchesStartedAt, max(deleted_at) as maxBatchesDeletedAt from batches'
-    ) as {
-      maxBatchesStartedAt: Iso8601Timestamp;
+
+    // Deleting non-empty batches will update the CVR count
+    const { maxBatchesDeletedAt } = this.client.one(`
+      select
+        max(batches.deleted_at) as maxBatchesDeletedAt
+      from batches inner join sheets
+      on sheets.batch_id = batches.id
+      where sheets.deleted_at is null
+    `) as {
```

Comment:
I update the logic a bit of what invalidates the backup. For example, I think deleting an empty batch should not invalidate a backup. Logic is laid out in the test file.